EpikCord/client/client.py

```diff
@@ -3,7 +3,27 @@
 
 
 class Client:
+    """The main class of EpikCord. Use this to interact with the Discord API and Gateway."""
     def __init__(self, token: str, intents: Intents, *, version: int = 10):
+        """
+        Parameters
+        ----------
+        token: str
+            The token of the bot.
+        intents: Intents
+            The intents of the bot.
```

Comment:
This is way too generic, no one want to read that

Reply:
Yeah I can be more specific with it. Although being that vague also isn't too bad, it's the token of your bot.

Reply:
This is a great place to remind people on how to retrieve the token, (maybe as an url to a side page), the importance of not sharing it etc. Boring text, but still preventive for the few that are sage enough to take a read through it.